pages/previewModel.js

```diff
@@ -86,11 +115,20 @@ function loadModel(gltf, resetCamera) {
 
         viewer._model = model;
         viewer.onLoad.raiseEvent(model);
-    }).otherwise(function(e){
+    }).otherwise(function(e) {
         window.alert('Error: ' + e);
     });
 }
 
-var gltf = JSON.parse(document.getElementById('glTF').textContent);
+var gltfFileName = document.getElementById('gltfFileName').textContent;
+var gltfRootPath = document.getElementById('gltfRootPath').textContent;
 
-loadModel(gltf, true);
+try {
+    var gltfContent = JSON.parse(document.getElementById('gltf').textContent);
+    loadModelFromContent(gltfContent, gltfRootPath, true);
+}
+catch (ex) {
+    // If the glTF content is missing or not valid JSON, then try to load the
+    // model directly from the glTF file.
+    loadModelFromFile(gltfFileName, gltfRootPath, true);
```

Comment:
Thanks for the ideas.  Looking into your first suggestion.  That would solve a separate problem that I'm having with the Issue #5 implementation, and it would also help with resolving Issue #1.  I'll iterate on this approach and try and get an update out for this PR within the next day or two.